### PR TITLE
Add docs for CSI Liveness Probe container image

### PR DESCRIPTION
**Pull request by croomes**
Added missing doc

### CODE REVIEW

**Comment by avestuk** [comment]
@croomes is it worth adding the tls config options to the table as well as I believe they are currently missing?

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ Parameter | Description | Default
 `images.csiClusterDriverRegistrarContainer` | CSI Cluster Driver Registrar Container image | `quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1`
 `images.csiExternalProvisionerContainer` | CSI External Provisioner Container image | `storageos/csi-provisioner:v1.0.1`
 `images.csiExternalAttacherContainer` | CSI External Attacher Container image | `quay.io/k8scsi/csi-attacher:v1.0.1`
+`ìmages.csiLivenessProbeContainer` | CSI Liveness Probe Container Image | `quay.io/k8scsi/livenessprobe:v1.0.1`
```
Comment (darkowlzz) [inline README.md]: We need to add this in the CRD as well `deploy/crds/storageos_v1_storageoscluster_crd.yaml` and `deploy/olm/storageos/storageoscluster.crd.yaml`.
I've found some tools like `yt`(yaml processor) to reduce this duplication. Have to automate replicating the changes in all the related files.

Comment (croomes) [reply README.md]: :) just noticed and pushed

Comment (croomes) [reply README.md]: @avestuk let's do the TLS options in a separate PR

File: pkg/controller/storageoscluster/storageoscluster_controller.go [hunk 2]
```diff
@@ -187,6 +187,7 @@ func (r *ReconcileStorageOSCluster) reconcile(m *storageosv1.StorageOSCluster) e
 		m.Spec.Images.CSINodeDriverRegistrarContainer = m.Spec.GetCSINodeDriverRegistrarImage(storageos.CSIV1Supported(r.k8sVersion))
 		if storageos.CSIV1Supported((r.k8sVersion)) {
 			m.Spec.Images.CSIClusterDriverRegistrarContainer = m.Spec.GetCSIClusterDriverRegistrarImage()
+			m.Spec.Images.CSILivenessProbeContainer = m.Spec.GetCSILivenessProbeImage()
```
Comment (darkowlzz) [inline pkg/controller/storageoscluster/storageoscluster_controller.go]: Thanks, this is important and was missing.

**Pull request merged**